### ISSUE TITLE
Double tap doesn't trigger neither OnMapClicked nor OnMapInteraction

### ISSUE THREAD

**Issue by MatteoColombo** [CartoDB/mobile-sdk#483]
We're trying to use a custom MapListener but we noticed that if you try to perform a double tap gesture to zoom in, neither OnMapClicked nor OnMapInteraction are called.

We also noticed that the map pivot mode is not respected, but I assume this is by design.
If you set it to PivotModeCenterpoint and then double tap, it doesn't zoom-in in the center of the map, but instead where you tapped .

**Comment by mtehver** [comment]
@MatteoColombo You are enabling 'zoom gestures' option of the SDK, right? In that case SDK indeed does not call `OnMapClicked` (as SDK handles the click action itself), but `OnMapInteraction` should work. I did a quick test using Java/Android, and it worked as expected.

Regarding pivot point, I think it would make sense to change the behavior when 'center point' mode is used. Using center point in case of double tap makes more sense. I will look into this.

**Comment by mtehver** [comment]
@MatteoColombo After reviewing `OnMapInteraction` handling, it seems you are correct in the sense that the callback is made before the zoom action has finished. Is this what you are experiencing?

**Comment by MatteoColombo** [comment]
Hi, yes zoom gestures are enabled.
here you can see what happens: https://youtu.be/62pDbUc174Y
Tap: OnMapClicked is called (expected)
Two fingers tap: OnMapInteraction is called with  IsZoom=true (expected)
Double tap: OnMapInteraction is not called, but the zoom level changes (not expected, expected)

I attached an example project with Xamarin forms and iOS

**Comment by MatteoColombo** [comment]
[DoubleTapDemo.zip](https://github.com/CartoDB/mobile-sdk/files/7926482/DoubleTapDemo.zip)

**Comment by mtehver** [comment]
@MatteoColombo Sorry for the delayed response and thanks for the demo app. I have now fixed this in 'develop' branch. The fix adds missing `OnMapInteraction` call in case of double tap. In addition, if pivot mode is set to `PivotModeCenterpoint`, then both double tap and dual click use center point as target zoom point.

In addition I added `AnimationStarted` property to `MapInteractionInfo`, it is set true if the interaction starts an animation. If false, then the interaction is performed before the callback.

The fix will be included in 4.4.5 release, current ETA is end of Q1.
